### PR TITLE
Add additional plans to bldr.toml

### PR DESCRIPTION
**Pull request by reset**
Add some additional build targets for bldr plans in the Habitat repo

* hab-studio
* hab-pkg-aci
* hab-pkg-export-docker
* hab-pkg-mososize
* hab-pkg-tarize

### CODE REVIEW

**Comment by thesentinels** [comment]
Thanks for the pull request! Here is what will happen next:
 1. Your PR will be reviewed by the maintainers
 2. If everything looks good, one of them will `approve` it, and your PR will be merged.

Thank you for contributing!

**Comment by reset** [comment]
@fnichol how's this look?

File: .bldr.toml [hunk 1]
```diff
@@ -136,3 +139,15 @@ paths = [
   "components/core/*",
   "components/builder-depot-client/*",
 ]
+
+[hab-pkg-aci]
+paths = ["components/pkg-aci/*"]
+
+[hab-pkg-export-docker]
+paths = ["components/pkg-export-docker/*"]
```
Comment (fnichol) [inline .bldr.toml]: You can also add:

* components/core/*
* components/common/*
* components/hab/* (soon to be dropped, I need to move a module over first)

Comment (reset) [reply .bldr.toml]: Will do!

**Pull request merged**